src/backend/api/reviews.js

```diff
@@ -0,0 +1,88 @@
+const { response } = require("express");
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+
+
+// GET - returns all  reviews
+router.get('/', async (req, res) => { 
+  try {
+    const review = await knex('reviews').select('*');
+    response.json(review);
+  } catch (error) {
+    res.status(500).send("error");
+    throw error;
+  }
+  
+});
+
+
+//POST - adds a new all Reviews  to the database
+
+router.post('/', async (req, res) => {
+  try {
+    const newReview = req.body;
+      const insertedReview = await knex('reviews').insert(newReview);
```

Comment:
You should check whether the new review is inserted, before sending the response.